### ISSUE TITLE
CLI: Refactor warning about .soliumignore not found

### ISSUE THREAD

**Issue by duaraghav8** [duaraghav8/Ethlint#249]
~The warning is annoying. `.soliumignore` should be completely optional, but still created when `--init` is run.~
~This is an extension of #176~

Change warning message to
```
[WARNING] .soliumignore not found. Use --no-soliumignore to make this warning go away.
```
if the issue is that .soliumignore file was not found

**Comment by duaraghav8** [comment]
Resolved in `v1.2.2`.
